woocommerce/woocommerce
# Remove inbox notes exceeding characters warning

### All Submissions:

### Changes proposed in this Pull Request:

Removes an old warning on inbox notes exceeding 320 character limit.

Also, removes `dompurify` from data package's dependency.



### How to test the changes in this Pull Request:

1. Go to WooCommerce > Home
2. Observe that inbox notes are rendered as usual
3. Open developer console and observe no errors or warning related to the change

### Other information:




### FOR PR REVIEWER ONLY:
